src/core_modules/capture-core/components/Pages/Search/SearchForm/SearchForm.component.js

```diff
@@ -45,12 +45,14 @@ const Index = ({
     searchViaUniqueIdOnScopeProgram,
     searchViaAttributesOnScopeProgram,
     searchViaAttributesOnScopeTrackedEntityType,
+    saveCurrentFormData,
     selectedSearchScopeId,
     classes,
-    availableSearchOptions,
+    searchGroupForSelectedScope,
```

Comment:
NIT: I would prefer this to be named searchGroup**s**ForSelectedScope, signaling there are multiple searchGroups here. This is nitpicking, I know, but this threw me off a bit.

Reply:
yeah thanks for this! :)